From 5cc8d1f96ab6d3459e13631d8356f87381562352 Mon Sep 17 00:00:00 2001
From: Jussi Kukkonen <jussi.kukkonen@intel.com>
Date: Thu, 2 Jun 2016 09:49:44 +0300
Subject: [PATCH] Continue even if fs size is not divisible by
 sectors-per-track

Filesystem with a size that is not divisible by sectors-per-track is
not broken or invalid: it might be unoptimized on media where
heads and cylinders actually matter but that should be it.

Preserve the warning but make mtools continue even if fs size is not
divisible by sectors-per-track.

Upstream-Status: Submitted [info-mtools@gnu.org]
Signed-off-by: Jussi Kukkonen <jussi.kukkonen@intel.com>
---
 init.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/init.c b/init.c
index c9152e0..4e16a01 100644
--- a/init.c
+++ b/init.c
@@ -338,7 +338,6 @@ Stream_t *fs_init(char drive, int mode, int *isRop)
 		fprintf(stderr,
 			"Add mtools_skip_check=1 to your .mtoolsrc file "
 			"to skip this test\n");
-		exit(1);
 	}
 
 	/* full cylinder buffering */
-- 
2.1.4

